features/cachecleaner/api/build.gradle.kts

```diff
@@ -0,0 +1,27 @@
+/*
+ * Copyright (c) 2023 New Vector Ltd
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+plugins {
+    id("io.element.android-library")
+}
+
+android {
+    namespace = "io.element.android.features.cachecleaner.api"
```

Comment:
I don't think it should be in a feature module, it could even live in an existing one (`androidutils`?).
Also I'd say we don't need an interface for that.

Reply:
Yeah as you said before the distinction between libraries & features subdirs is kinda moot, we might as well merge them into a single subdir.

This is definitely not a utility class though because it is not general purpose, it is not something that could be reused easily in another context. So I wouldn't put it in `androidutils`.

For the interface yes, we could do without it... but it's there and we've already spent enough time for such a simple thing. I hope it won't bother anyone.